### DISCUSSION TITLE
Ultimate Blog Update!

### DISCUSSION THREAD

**Discussion by barbarbar338** [barbarbar338/webpage#12] [Posts]
## New design, new back-end and a NEW BLOG!
The update that I have been thinking about for a long time is with you from today!

As you know, a while ago (about 4 months 😃) I posted an article titled "New Blog!" and it was pretty exciting. However, from the moment I shared that article, I regretted it. Because the blog I shared the article was under another domain name and its design was not compatible with my current site at all. Also the backend was very bad and very difficult to manage. On top of that, you see this work that took a long time and you read this article. Say hello to my **NEW BLOG!**

### Technology update on back-end
If you have read my previous post, you must have understood how lazy I am 😃. Of course, I didn't bother to write a separate back-end for sharing and managing posts. I decided to use [GitHub Discussions](https://docs.github.com/en/discussions) instead! Thus, my posts will be managed entirely via GitHub and it will be very easy for me to edit them (updating the posts and updating the tags).

Another benefit of [GitHub Discussions](https://docs.github.com/en/discussions) is that it made it easier for me to integrate the comment system into posts. Since we can comment on discussions, I have added the comment system by showing these comments under my posts. And I did this with an application called [Giscuss](https://giscus.app/). Giscuss is a comments system powered by [GitHub Discussions](https://docs.github.com/en/discussions). It lets visitors leave comments and reactions on your website via [GitHub](https://github.com/)!

### Front-end update
The blog is now linked to the main site and its design matches the main site. Nothing much. 😬

### And as always...
Thank you for reading this far and spending your valuable time. Hope to see you again in a similar article in the future... 😍

**Comment by barbarbar338** [comment]
Wow, this comment thing looks cool!

**Reply by fishuke** [reply]
its not cool, its fucking cool

**Comment by fishuke** [comment]
This is neat!

**Comment by eggsy** [comment]
Nice.
